docs/reference/files/tests-integration-test-charm-py-file.rst

```diff
@@ -0,0 +1,52 @@
+.. _tests-integration-test-charm-py-file:
+
+
+``tests/integration/test_charm.py`` file
+========================================
+
+The ``tests/integration/test_charm.py`` file is the companion to
+``src/charm.py`` for integration testing.
+
+This file is created automatically by ``charmcraft init`` and it is
```

Comment:
Not all profiles generate this file.

Reply:
Let's change it to "profiles other than the one for 12-factor apps automatically create..."